src/Modules/KnowledgeComponents/Tutor.KnowledgeComponents.Core/Domain/KnowledgeMastery/SessionTracker.cs

```diff
@@ -44,19 +54,28 @@ public Result Terminate()
 
     public Result Pause()
     {
-        if (!HasUnfinishedSession) Launch();
+        if (!HasUnfinishedSession) return Result.Fail("No active session to pause.");
```

Comment:
In the scenario where people have left their browser running and shutdown their PC - the pause event triggers as soon as the PC is started? Will it have an active session?

What if they opened the Tutor in another app (e.g., mobile) in the meantime? The other session should activate, abandoning this one? Verify that such a scenario cannot lead to errors in events.